src/routes/home/home.js

```diff
@@ -40,6 +41,13 @@ export default function home(props) {
       <Button variant='contained' onClick={() => handleRedirectToStatistics()}>
         Click me to go to Statistics page
       </Button>
+
+      <FirebaseContext.Consumer>
+        {firebase => {
+          console.log(firebase.testLog());
```

Comment:
`testLog()` is a void function that already logs, here you'd just be logging nothing